recipe-server/normandy/recipes/tests/test_serializers.py

```diff
@@ -168,6 +168,42 @@ def test_validation_with_valid_data(self):
         }
         assert serializer.errors == {}
 
+    def test_update_action(self):
```

Comment:
Yeah sure, I wrote these when I was trying to isolate where the issue was occuring. I just kept them in cause they were already written. But I can ditch 'em.